google/cloud/storage/internal/grpc/default_options_test.cc

```diff
@@ -120,12 +121,22 @@ TEST(DefaultOptionsGrpc, DefaultOptionsUploadBuffer) {
   EXPECT_EQ(with_override, 256 * 1024L);
 }
 
+#ifdef GRPC_CPP_VERSION_MAJOR
 TEST(DefaultOptionsGrpc, MetricsEnabled) {
   ScopedEnvironment env("CLOUD_STORAGE_EXPERIMENTAL_GRPC_TESTBENCH_ENDPOINT",
                         absl::nullopt);
   auto const options = DefaultOptionsGrpc(Options{});
-  EXPECT_TRUE(options.get<storage_experimental::EnableGrpcMetricsOption>());
+  auto const expected =
+      (GRPC_CPP_VERSION_MAJOR >= 1 &&
+       (                                                                    //
+           (GRPC_CPP_VERSION_MINOR == 63 && GRPC_CPP_VERSION_PATCH > 1) ||  //
+           (GRPC_CPP_VERSION_MINOR == 64 && GRPC_CPP_VERSION_PATCH > 1) ||  //
+           (GRPC_CPP_VERSION_MINOR >= 65)                                   //
+           ));
```

Comment:
I suggest defining `GrpcEnableMetricsIsSafe` in the header, then calling it here.

Reply:
Done